app/graph/mutations/tasks_order_mutation.rb

```diff
@@ -0,0 +1,14 @@
+TasksOrderMutation = GraphQL::Relay::Mutation.define do
```

Comment:
Why this and not the usual `set_fields` to set the order via the regular `UpdateTaskMutation` ? @caiosba

Reply:
oh, I see this was specified in the ticket - sorry